src/app/(space)/(content)/[[...pathname]]/PageClientLayout.tsx

```diff
@@ -9,5 +12,25 @@ export function PageClientLayout(props: {}) {
     // We use this hook in the page layout to ensure the elements for the blocks
     // are rendered before we scroll to the hash.
     useScrollToHash();
+
+    // When the user switches variants using the space dropdown, we pass a fallback=true parameter.
+    // This parameter indicates that we should redirect to the root page if the path from the
+    // previous variant doesn't exist in the new variant. If the path does exist, no redirect occurs,
+    // so we need to remove the fallback parameter.
```

Comment:
Not important: It find it better to document/comment at the function definition level, as it can be visualized in VS code and also more accessible in cases of multiple uses.